tests/test_inference.py

```diff
@@ -0,0 +1,18 @@
+from cities.modeling.model_interactions import InteractionsModel
+
+
+# TODO this needs to be parametrized by outcome datasets,
+# intervention datasets and forward shifts
+def test_InteractionsModel():
```

Comment:
sometimes I'll call these "..._runtime_test" explicitly just so I don't think it's testing correctness.